api/src/main/java/io/opentelemetry/common/AttributeValue.java

```diff
@@ -325,12 +341,13 @@ public final Type getType() {
   @AutoValue
   abstract static class AttributeValueStringArray extends AttributeValue {
 
+    private static final List<String> EMPTY = Collections.<String>emptyList();
+
     AttributeValueStringArray() {}
 
     static AttributeValue create(String... stringValues) {
       if (stringValues == null) {
-        return new AutoValue_AttributeValue_AttributeValueStringArray(
-            Collections.<String>emptyList());
+        return new AutoValue_AttributeValue_AttributeValueStringArray(EMPTY);
```

Comment:
could we just return a static final AttributeValue instance, rather than creating a new one for each call?

Reply:
I've overlooked a simple thing like that :sweat_smile:

Reply:
hey, that's why we review each others code. :). Easy to miss stuff like this when you're in the weeds.